src/module.js

```diff
@@ -59,33 +59,21 @@ var debug = {
     FeatureSelection
 };
 
-// Window can only be set in main thread
-if (Utils.isMainThread) {
-
-    window.Tangram = module.exports = {
-        leafletLayer,
-        debug,
-        version: version.string
-    };
-
-}
-
-if (Utils.isWorkerThread) {
```

Comment:
Yes that's right, the worker is loaded there. Background on how/why/alternatives here: https://github.com/tangrams/tangram/issues/252